pkg/apis/delivery/v1alpha1/policy_validation.go

```diff
@@ -26,5 +26,43 @@ import (
 // Validate implements apis.Validatable
 func (p *Policy) Validate(ctx context.Context) *apis.FieldError {
 	logging.FromContext(ctx).Infof("Validate called for %v", *p)
-	return nil
+	var err *apis.FieldError
+	// validate that the mode value must be "time" ("request" and "error" not supported for now)
+	if p.Spec.Mode != "time" {
+		err = err.Also(apis.ErrGeneric("Invalid mode value (accepted values: 'time')", "spec.mode"))
```

Comment:
we have ErrOneOf, probably better here.

Reply:
I see. I ended up using ErrInvalidValue for this one though (the ErrOneOf feels a bit odd here)